test/supertest.js

```diff
@@ -150,7 +154,7 @@ describe('request(app)', function(){
 
     request(app)
     .get('/')
-    .expect(302, done);
+    .expect(302, function() { done(); });
```

Comment:
ok, that totally makes sense. I think what you have is fine. I am thinking maybe we just update the supertest [README](https://github.com/visionmedia/supertest/blame/master/Readme.md#L48) talking about this feature to make special note of this.